src/lib.rs

```diff
@@ -259,6 +272,9 @@ mod tests {
         assert_eq!(plaintext, data);
     }
 
+    // We don't run this test with mock-crypto because it depends on enabling real crypto features
+    // to use precomputed keys.
+    #[cfg(not(feature = "use-mock-crypto"))]
```

Comment:
why ? we can do it via [review comment to this](https://github.com/maidsafe/safe_crypto/pull/5/files#diff-09b0eb261ff3c84c9864b11431935b45R150) - or will that be a problem somehow ?